pkg/cfn/builder/vpc.go

```diff
@@ -30,8 +31,10 @@ const (
 	PrivateRouteTableAssociation = "RouteTableAssociationPrivate"
 	PubSubRouteKey               = "PublicSubnetDefaultRoute"
 	PubSubIPv6RouteKey           = "PublicSubnetIPv6DefaultRoute"
+	PubSubDNS64IPv6RouteKey      = "PublicSubnetDNS64IPv6DefaultRoute"
```

Comment:
This is unused. Were you planning to use it somewhere?

Reply:
Good spot, that can be removed!